angular-ngrx.js

```diff
@@ -6,18 +6,41 @@ module.exports = {
     {
       files: TYPESCRIPT_FILE_PATTERNS,
 
-      extends: [
-        '@code-pushup/eslint-config/angular',
-        'plugin:@ngrx/recommended-requiring-type-checking',
-      ],
+      plugins: ['@ngrx'],
+      extends: ['@code-pushup/eslint-config/angular'],
 
       rules: {
         // CUSTOMIZED RULES FROM EXTENDED CONFIGS
-
+        '@ngrx/avoid-combining-component-store-selectors': 'warn',
         '@ngrx/avoid-cyclic-effects': 'error',
         '@ngrx/avoid-duplicate-actions-in-reducer': 'error',
```

Comment:
I took the [original config from version 17.2](https://github.com/ngrx/platform/blob/17.2.0/modules/eslint-plugin/src/configs/recommended-requiring-type-checking.ts) as a reference

Reply:
I understand. Then I would prefer to keep those 2 rules as errors. The idea is to use errors for potential bugs, and warnings for code style issues. Since [`avoid-cyclic-effects`](https://ngrx.io/guide/eslint-plugin/rules/avoid-cyclic-effects) guards against infinite loops, and [`avoid-duplicate-actions-in-reducer`](https://ngrx.io/guide/eslint-plugin/rules/avoid-duplicate-actions-in-reducer) is generally a mistake by the programmer, so I thought `error` was a more appropriate severity level.